### PR TITLE
The Grand Weedchurch

### PR DESCRIPTION
**Pull request by terezi4real**
## About the PR

Adds The WeedChurch! a great structure built to commemorate the God of the 420th dimension, as well as give its followers a place to partake in the holy act of "getting high"
## Technical details
PR only adds map, i dunno how to make it a PoI, i found the file but then my eyes turned a milky white and i had to go outside and feel a trees bark to return to normalcy.
## Media
pixelated ass (and minorly outdated) image. map renderer no worky

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added the WeedChurch

### CODE REVIEW

**Comment by Cheackraze** [comment]
im liking what i see so far in the picture, will have to wait till im home till i can test it in-game too. You'll want to make sure it has things like becomesstation component and such, since this will be acting more like a station than our smaller POI.  Design wise, from just a brief glance, i would say try to squeeze in another dock or two here:
image

I also added the church items just a bit ago, so feel free to map those too.  Also, i can tell your central gardening area, i like the idea, but i think it needs more pizzazz, more work to make that central communal area really shine, but im willing to open it up and check it out and think of some stuff for it too

File: Resources/Prototypes/Maps/weedchurch.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      stationProto: StandardFrontierStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Frontier Outpost {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
```
Comment (Cheackraze) [inline Resources/Prototypes/Maps/weedchurch.yml:18]: this makes the station have infinite open job slots, and round start even. I'm not necessarily opposed to having new round start locations, but we should hold off for now until we can chat on discord about what a station might need to be a round start spawn point

**Review by Cheackraze — CHANGES_REQUESTED** [review]
im gunna open this up and review it in game so we can get final changes done

File: Resources/Prototypes/Maps/weedchurch.yml (hunk 1 above)
Comment (terezi4real) [reply Resources/Prototypes/Maps/weedchurch.yml:18]: Was for testing purposes, im fine with no roundstart spawns. but yeah HMU and we can talk

**Pull request closed without merging**